### ISSUE TITLE
CASA-Wide Editorial WG Mtg - 26 Oct

### ISSUE THREAD

**Issue by bumblefudge** [ChainAgnostic/CASA#95]
# CASA Editorial 26 Oct

## Announcements
- WalletUnCon planning proceeding apace, surprisingly few cancellations, all organizers still planning to attend at time of writing - massive backlog of tickets but still working out waves of ticketing to minimize empty seats

## PRs to refine/move to close
- Need 2nd review from anyone confident but LGTM
    + [Namespaces #86 - Monero/CAIP-10](https://github.com/ChainAgnostic/namespaces/pull/86) - this one is urgent and important, please halp!
    + [Namespaces #91 - mina](https://github.com/ChainAgnostic/namespaces/pull/91)
    + [Namespaces #99 - multiversX](https://github.com/ChainAgnostic/namespaces/pull/88)
- Need 2nd review specifically from the CACAO specialists (@ukstv, @haardikk21, @chunningham, @oed)
    + [Namespaces #84 - XRPL/CAIP-122](https://github.com/ChainAgnostic/namespaces/pull/84)
    + [CAIP minute process update to contributing.md](https://github.com/ChainAgnostic/CAIPs/pull/246) - just a little vitamin since i'm noticing PRs never seem to notice the conventions
    
## Ongoing projects/topics
- Big news: CAIP-222 breakage and/or iteration in [CAIPs#248](https://github.com/ChainAgnostic/CAIPs/pull/248) - slight change to 222 which allows dapp to offer a SIWX template for *multiple* chainIds and/or signature types 
    - example use-case: "expose to me an address on ANY of these chains to sign in" OR "sign this SIWX message the EIP-1193 way or the EIP-1273 way but not the EIP-712 way"
    - response includes an ARRAY OF cacaos rather than one (also helpful for chaining CACAOs in case of possible complex delegation/UCAN-signing scenarios)
    - the big question is, break 222 with legacy support, break 222 without legacy support, or just make 248 a separate spec to simplify things for those just now tuning in?
        - important: we need to decide this before WalletUncon so that we can have a workshop on use-cases and UX for it :sweat_smile: @chunningham @oed @awoie any input appreciated
- New/open and waiting on more review
    - [Errata arweave/caip122](https://github.com/ChainAgnostic/namespaces/pull/97) - anyone know how to tag/track down @ropats16, @joshbenaron @DanMacdonald or @Zachferland ? this is debatably a non-breaking change but I'd still rather get an implementer to confirm it's NBD
    - [namespaces #90/Aleo](https://github.com/ChainAgnostic/namespaces/pull/90) - author @jonandgon's approval needed for minor tweaks before mergeable
    - [CAIPs#238](https://github.com/ChainAgnostic/CAIPs/pull/238) - semver/feature-discovery handshake format? this was opened by an IaaS co that needs this as both painkiller AND vitamin, so anyone in AllCoreDevs or Lightclient or node-aaS/NodeDevOps communities welcome to chime in!
- Varsig/multiformats
    - libp2p team at PL wanted to use varsig for some decentralized handshake protocols in the libp2p family, but were proposing drastically different varsig prefix sequence and new multiformats key representations. stay tuned for cagefights
        + see also new issue [#12](https://github.com/ChainAgnostic/varsig/issues/12) for details on possible rejiggering/breakage of varsig syntax 
    - webAuthN extension weeeeen
    - [CAIPs#234] - ongoing issue with custom/extra UCAN headers in varsig and/or in CACAOs (no easy answer, still mulling on this)
- Issues that are still open that the editors want outside help with
    + [Namespaces #87](https://github.com/ChainAgnostic/namespaces/issues/87) - BTC/CAIP-10 examples don't match text (and generally need a post-Taproot/Ordinals update, clarity on which address types are out of scope or differently-capable, etc)
    + [Namespaces #56](https://github.com/ChainAgnostic/namespaces/issues/87) - Still need help updating CAIP2 and 10 now that XCMv3 is in place, if this stays open another 6 months we will write a small Web3 Foundation grant and do it ourselves so help me Gaia

## Backlog
- [CAIPs#225](https://github.com/ChainAgnostic/CAIPs/issues/225) - i18n for SIWX and SIWE???
